hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieMergeHandle.java

```diff
@@ -147,6 +149,13 @@ public HoodieMergeHandle(HoodieWriteConfig config, String instantTime, HoodieTab
     this.preserveMetadata = true;
     init(fileId, this.partitionPath, dataFileToBeMerged);
     validateAndSetAndKeyGenProps(keyGeneratorOpt, config.populateMetaFields());
+    // The compactor avoids heavy rewriting when copy the old record from old base file into new base file
+    if (config.populateMetaFields()) {
+      LOG.info("Using update instead rewriting during compaction");
```

Comment:
Set the log as debug level, "instead" -> "instead of".

Reply:
> Set the log as debug level,